auto-completion-form

From: Bryan Larsen <bryan@larsen.st>

Now for the form in `projects/show.dryml`. We'll use Hobo's ajax `part` mechanism to refresh the collection without reloading the page:

SHOW_PATCH

Some things to note:

 - The `<collection>` tag has `part="members"`. This creates a re-loadable section of the page, much as you would achieve with partials in regular Rails.

 - The `<form>` tag has `update="members"`. The presence of this attribute turns the form into an ajax form. Submitting the form will cause the "members" part to be updated.

 - The `<name-one>` tag creates an input field for the user association with auto-completion.
---

 app/views/projects/show.dryml |   13 +++++++++++++
 1 files changed, 13 insertions(+), 0 deletions(-)


diff --git a/app/views/projects/show.dryml b/app/views/projects/show.dryml
index 2e397ff..85d9355 100644
--- a/app/views/projects/show.dryml
+++ b/app/views/projects/show.dryml
@@ -9,4 +9,17 @@
     <empty-message:>No stories match your criteria</empty-message:>
   </table-plus>
   </collection:>
+
+  <aside:>
+    <h2>Project Members</h2>
+    <collection:members part="members"/>
+
+    <form:memberships.new update="members" reset-form refocus-form>
+      <div>
+        Add a member:
+        <name-one:user/>
+      </div>
+    </form>
+  </aside:>
+
 </show-page>
\ No newline at end of file
